tc/sgx/trusted_worker_manager/enclave/work_order_preprocessed_keys.h

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#pragma once
+
+#include <string>
+
+#include "types.h"
+#include "work_order_data.h"
+#include "enclave_data.h"
+
```

Comment:
Please add a comment what this class is used for

Reply:
Done